module.mk

```diff
@@ -35,10 +35,6 @@ else
 $(error Unknown value for CONFIG "$(CONFIG)")
 endif
 
-ifeq ("$(ARCH_NAME)", "blackhole")
-$(error Blackhole support not yet added!)
```

Comment:
Are there no changes to cmake needed?

Might want to put an error messages in blackhole as well, however that'll break the build pipelines.

Reply:
I didn't see any errors when compiling with blackhole through cmake flow, why would we want error messages there?

Reply:
Never mind I'm an idiot, read this as an addition not deletion